### PR TITLE
Add new device ClearSky Airlink 4G/LTE telemetry

### PR DESCRIPTION
**Pull request by AlexandrShipovsky**
Hello!
I just add a new device into wiki. Also i did a local build and made sure there were no errors.
Links on overview device:
[https://www.youtube.com/watch?v=XszxSj__-Xg](url) 
[https://air-link.space/](url)

### CODE REVIEW

File: common/source/docs/common-airlink-telemetry.rst [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+.. image:: ../../../images/clearsky-airlink/airlink.png
+
+Overview
+========
+
+`ClearSky Airlink <https://air-link.space>`__ very small 4G/LTE telemetry module. It allows you to control a drone or an airplane via the Internet from anywhere on earth.
+
+Features
+========
+* 1. Telemetry command and control.
```
Comment (Hwurzburg) [inline common/source/docs/common-airlink-telemetry.rst:16]: remove * and numeral... and replace with -   etc.
```suggestion
- Telemetry command and control.
```

File: common/source/docs/common-airlink-telemetry.rst [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+* 2. Send data to `Flightradar24 <https://www.flightradar24.com>`__ and `Open Glider Network <https://www.glidernet.org/>`__ (ADS-B).
+* 3. Multiple vehicles control (test).
```
Comment (Hwurzburg) [inline common/source/docs/common-airlink-telemetry.rst:18]: I expect the "(test)" means that its still being tested....if so:
```suggestion
- Multiple vehicles control (in development).
```

File: common/source/docs/common-telemetry-landingpage.rst [hunk 3]
```diff
@@ -55,6 +55,7 @@ Long Range
     SPL Satellite Telemetry <https://discuss.ardupilot.org/t/stretching-comm-links-from-indoors-to-the-globe/45896>
     UAVCast 3G/4G Cellular <common-uavcast-telemetry>
     XBStation 4G LTE Link <common-xbstation-telemetry>
+    ClearSky Airlink 4G LTE Telemetry <common-airlink-telemetry>
```
Comment (Hwurzburg) [inline common/source/docs/common-telemetry-landingpage.rst:58]: should be listed in alphabetical order

**Review by Hwurzburg — CHANGES_REQUESTED** [review]
a few changes needed.....very nice device....if you need a US tester, let me know :)

**Comment by AlexandrShipovsky** [comment]
> a few changes needed.....very nice device....if you need a US tester, let me know :)

Thank you! Unfortunately, the bands are only suitable for Europe, but the next version will be global and we will definitely send it to you)))
I fixed everything, please check

**Comment by Hwurzburg** [comment]
I rebased and squashed....merging

**Pull request merged**